### PR TITLE
Add try_deploy_local & improve readability

### PR DESCRIPTION
**Pull request by wootmatt**
* Improved the overall readability of the codebase 
* Added try_deploy_local, so that those who have downloaded a binary do not have to do so again

### CODE REVIEW

File: src/actor.rs [hunk 1]
```diff
@@ -109,7 +124,7 @@ impl<'a> Actor<'a> {
     pub fn deploy_remote(&self, git_location: &str, file_name: &str) -> Result<Actor, Error> {
```
Comment (rohanp9000) [inline src/actor.rs:124]: Can we change this param from git_location to remote_location?

File: src/actor.rs [hunk 2]
```diff
@@ -67,6 +67,21 @@ impl<'a> Actor<'a> {
         Ok(())
     }
 
+    /// Attempts to deploy a program if it exists locally. If it does not,
+    /// it will fall back on deploy_remote.
+    pub fn try_deploy_local(
+        &self,
+        program_location: &std::path::Path,
+        fallback_git_location: &str,
```
Comment (rohanp9000) [inline src/actor.rs:75]: Here as well - fallback_git_location -> fallback_remote_location

Comment (wootmatt) [reply src/actor.rs]: Done

File: src/actor.rs (hunk 1 above)
Comment (wootmatt) [reply src/actor.rs]: and done

**Review by yuhanfang — APPROVED** [review]

**Pull request merged**